test/ambuda/conftest.py

```diff
@@ -152,3 +169,17 @@ def admin_client(flask_app):
     session = get_session()
     user = session.query(db.User).filter_by(username="akprasad").first()
     return flask_app.test_client(user=user)
+
+
+@pytest.fixture()
+def deleted_client(flask_app):
+    session = get_session()
+    user = session.query(db.User).filter_by(username="sandrocottus").first()
+    return flask_app.test_client(user=user)
+
+
+@pytest.fixture()
+def banned_client(flask_app):
+    session = get_session()
+    user = session.query(db.User).filter_by(username="sikander").first()
```

Comment:
it's more boring, but for the sake of helping understand the code, maybe include some ID like `-banned` or `-deleted` in the username